pkg/plugins/common/kustomize/v2/scaffolds/webhook.go

```diff
@@ -71,6 +71,21 @@ func (s *webhookScaffolder) Scaffold() error {
 		return fmt.Errorf("error updating resource: %w", err)
 	}
 
+	if err := scaffold.Execute(
+		&kdefault.WebhookCAInjectionPatch{},
+		&kdefault.ManagerWebhookPatch{},
+		&webhook.Kustomization{Force: s.force},
+		&webhook.KustomizeConfig{},
+		&webhook.Service{},
+		&certmanager.Certificate{},
+		&certmanager.Kustomization{},
+		&certmanager.KustomizeConfig{},
+		&patches.EnableWebhookPatch{},
+		&patches.EnableCAInjectionPatch{},
+	); err != nil {
+		return fmt.Errorf("error scaffolding kustomize webhook manifests: %v", err)
+	}
```

Comment:
We just putting the execution here to occur before we start uncomment the pacthes and etc. We did not change this one. Just added

&patches.EnableWebhookPatch{},
&patches.EnableCAInjectionPatch{},

which was in the CRD and should be only scaffold when we generate a webhook